src/Common/utils.ts

```diff
@@ -85,13 +85,13 @@ abstract class Utils {
     }
 
     // Xpring Common JS's API takes a string|undefined while the underlying address library wants string|false.
-    const shimTagParameter = tag !== undefined ? tag : false
+    const shimTagParameter = tag ?? false
```

Comment:
There's [an ESLint rule](https://eslint.org/docs/rules/no-negated-condition) against `if(negative) {...} else {...}`, because there's usually a better way to write it (like here).